src/components/Reviews/Reviews.jsx

```diff
@@ -9,31 +9,70 @@ export default function Reviews() {
     const [loading, setLoading] = useState(true);
     const [reviewsList, setReviewsList] = useState([]);
     const { category } = useParams();
+    const [sortBy, setSortBy] = useState("date");
+    const [orderBy, setOrderBy] = useState("asc");
+
 
     useEffect(() => {
         setLoading(true);
 
-        getReviews(category).then((response) => {
+        getReviews(category, null, sortBy, orderBy).then((response) => {
             setLoading(false);
             return setReviewsList(response.reviews);
         });
-    }, [category])
+    }, [category, sortBy, orderBy])
+
     
     if (loading) {
         return (<h2>Loading...</h2>)
     } else {
         return (
+          <section>
+            <label htmlFor="sortBy">Sort By: </label>
+            <select
+              id="sortBy"
+              onChange={(e) => setSortBy(e.target.value)}
+              value={sortBy}
+            >
+              <option key="date" value="date">
```

Comment:
Sort by comments?

Reply:
Sort by comments does not work as it is not implemented in the back end. This will can be resolved later but I don't want to spend loads of time playing around with it right now, I'd rather get some of the other tasks done and come back to revisit this in my own time.